### ISSUE TITLE
[1.0 Bughunt] In shaded view mode, closed sketches disappear in all workbenches

### ISSUE THREAD

**Issue by raindropsfromsky** [Ondsel-Development/FreeCAD#91]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

1. In PD WB, press V,6 (or use the _View > Draw style > Shaded_ menu).
2. Create a sketch. Select any plane.
2. Draw any shape (e.g. rectangle)
3. Close the Sketch. The sketch disappears.
4. Press v,1 (or use the _View > Draw style > As is_ menu.) The sketch reappears
5. Repeat v,1 and v,6 in all workbenches. The behavior is consistent across all workbenches.

**Desired behavior:** 
Once a sketch is closed, it should be treated distinctly from a 3D object.
In other words, it should not be treated as a collection of edges and points.
So a v,6 operation should not make it disappear.

### Full version info

```shell
OS: Windows 11 build 22631
Word size of Ondsel: 64-bit
Version: 2024.2.1.37226 (Git)
Build type: Release
Branch: (HEAD detached at 2024.2.1)
Hash: 1a1bd7290494aef925a026c87b808ac58d53c3b0
Python 3.11.9, Qt 5.15.13, Coin 4.0.2, Vtk 9.2.6, OCC 7.7.2
Locale: English/India (en_IN)
Installed mods: 
  * 3DfindIT 1.2.0
  * Ondsel-Lens 2024.3.26.01
  * SearchBar 1.0.2
```

### Subproject(s) affected?

None

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by lt72884** [comment]
I can confirm this as well in sketcher workbench, and part design

**Comment by raindropsfromsky** [comment]
> I can confirm this as well in sketcher workbench, part workbench and part design

I can see the disappearance only in the PD WB.
When I am in the Sketcher WB, the sketch does _not_ disappear.

I tried three possible workflows, and found no problem in the Sketcher WB:
1. In a new file, switch to Sketcher WB directly. Draw a sketch. Press v,6.
2. In a new file, start with PD WB, and then create a sketch. (Now we are in the Sketcher WB.) Draw a sketch. Press v,6.
3. I already have a 3D object in the PD WB. double-click on the **Sketch** node in the tree panel, the sketch opens normally, and it is fully visible.

I cannot confirm a problem in the Part WB.
While using the Part WB, I could not add a sketch.
I created a 3D object in the PD WB, which remains fully visible if I press v,6.
